package.json

```diff
@@ -26,7 +26,7 @@
     "postlint-styles-fix": "prettier --ignore-path .eslintignore --single-quote --write '**/*.{css,scss}'",
     "precommit": "lint-staged",
     "prestart": "yarn build",
-    "start": "cross-env NODE_ENV=production electron ./app/",
+    "start": "cross-env NODE_ENV=production electron ./app/main.prod.js",
```

Comment:
Worth noting, `./` can also be used instead, since it'll search the `main` specified in this `package.json`.

Reply:
Yup, that's right. I added it like so because I think most users of the boilerplate are usually new to node and probably wouldn't know that. I thought this was more explicit. Thoughts?

Reply:
Sounds good to me.